cpp/src/arrow/compute/test_util.h

```diff
@@ -82,6 +83,26 @@ struct DatumEqual<Type, enable_if_integer<Type>> {
   }
 };
 
+static inline void CheckScalarUnary(std::string func_name,
```

Comment:
Shouldn't this be put in a `test_util.cc`?

Reply:
Yes it should, will do